waddrmgr/db.go

```diff
@@ -902,35 +927,48 @@ func deserializeImportedAddress(row *dbAddressRow) (*dbImportedAddressRow, error
 		dbAddressRow: *row,
 	}
 
-	pubLen := binary.LittleEndian.Uint32(row.rawData[0:4])
+	pkHashLen := binary.LittleEndian.Uint32(row.rawData[0:4])
```

Comment:
Reminder to take care of this.  I don't particularly care for offset variables, but to be consistent with the rest of the code, it should create the offset var first, and then slice between `[offset:offset+fieldlen]`.